CHANGES.rst

```diff
@@ -6,6 +6,11 @@ documentation
 
 - Remove references to pub server [#7421]
 
+wfss_contam
+-----------
+
+- Open image models in with context to keep models open while accessing contents [#7425]]
```

Comment:
Perhaps put "with" in quotes, so folks know it's the name of an actual python code element (as opposed to a typo in the message!). There's also an extra "]" at the end of the line.

Oh, and this is not your fault, but the log file obviously didn't get updated to account for the fact that 1.9.2 got released on Wednesday. So can you give the 1.9.2 section a date of 2023-01-04, start a new section above it for 1.9.3 (marked as unreleased), and finally move this entry to the 1.9.3 section.

Reply:
Thanks for looking it over. Sorry about the extra "]".

I made the updates to the changelog. Please let me know if they look ok.

Reply:
Looks good. Thanks!